src/Simplic.Icon.UI/View/IconEditor.xaml.cs

```diff
@@ -82,13 +105,27 @@ public IconViewModel SelectedIcon
             }
         }
 
+        /// <summary>
+        /// Gets if an icon is selected.
+        /// </summary>
+        public Visibility VisibleIconSelected
+        {
+            get
+            {
+                return (this.DataContext as IconEditorViewModel).IsSelectedIcon;
+            }
+        }
+
         private void ListView_MouseDoubleClick(object sender, System.Windows.Input.MouseButtonEventArgs e)
         {
             if (SelectedIcon != null)
             {
-                this.WindowMode = WindowMode.View;
-                this.DialogResult = true;
-                this.Close();
+                if (this.DialogResult != null)
+                {
+                    this.WindowMode = WindowMode.View;
+                    this.DialogResult = true;
+                    this.Close();
+                }
```

Comment:
why do you want to close the window on a double klick on an icon?

Reply:
This is the part where the user selects an item and the window is closed after the selection.
This was implemented before I changed it.